dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -4079,6 +4079,12 @@ Sedp::populate_discovered_reader_msg(
     drd.readerProxy.associatedWriters.length(len + 1);
     drd.readerProxy.associatedWriters[len] = *writer;
   }
+  drd.ddsSubscriptionData.type_consistency.kind = DDS::ALLOW_TYPE_COERCION;
+  drd.ddsSubscriptionData.type_consistency.ignore_sequence_bounds = true;
+  drd.ddsSubscriptionData.type_consistency.ignore_string_bounds = true;
+  drd.ddsSubscriptionData.type_consistency.ignore_member_names = false;
+  drd.ddsSubscriptionData.type_consistency.prevent_type_widening = false;
+  drd.ddsSubscriptionData.type_consistency.force_type_validation = false;
```

Comment:
@ClaytonCalabrese and @iguessthislldo, These values are the defaults from the spec. Are they also the correct fixed/const values we want to use for now?

Reply:
Ignore member names is screwed up in the spec and I wrote an issue about that in our doc for spec problems.  The others make sense to me.

Reply:
Are you saying that the default for `ignore_member_names` should be `true`? What about the constant value we we wish to use for `ignore_member_names` for our initial setup?